### PR TITLE
Added IHP swag opportunity in data.json

### LINKED ISSUE #675: IHP - Integrated Haskel Platform
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : IHP (Integrated Haskel Platform)

Q. Where have you seen the announcement (URL)?  
A. : https://github.com/digitallyinduced/ihp#ihp-stickers
 

Q. What kind of swag (clothing, stickers, ...)?  
A. : stickers

Fill the form or write an email to hi@digitallyinduced.com

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by simranchanna**
closes #675

- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -242,6 +242,15 @@
     "dateAdded": "2019-10-02T21:06:00.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers", "expired"]
   },
+  {
+    "name": "IHP",
+    "difficulty": "easy",
+    "description": "Just fill out the Google form and get stickers via physical mail!",
```
Comment (vikaspotluri123) [inline data.json:248]:
```suggestion
    "description": "Fill out the form, and get some stickers sent to you!",
```

**Comment by vikaspotluri123** [comment]
@simranchanna FYI - you can link your PR to an issue by referencing the issue number (e.g. `#123`), and you can have it autoclose by adding `fixes` or `closes` before it (e.g. `closes #123` or `fixes #123`)

File: data.json [hunk 2]
```diff
@@ -242,6 +242,15 @@
...
+    "reference": "https://github.com/digitallyinduced/ihp#ihp-stickers",
+    "image": "https://lh4.googleusercontent.com/0LwDlgsPryo_Du5jkpCgrIPxnO5qNZ3HQtPSt62jYO1kiUBkKDafcgc2MBNF6T_7n_IDsvP0REmNOn3dmv1xvNN00hLnDXu2WNEvjUjvExqy9nbyYOD7uHfct7nW=w376",
+    "dateAdded": "2020-10-02T9:59:00.000Z",
```
Comment (vikaspotluri123) [inline data.json:251]:
```suggestion
    "dateAdded": "2020-10-02T09:59:00.000Z",
```

File: data.json [hunk 3]
```diff
@@ -242,6 +242,15 @@
     "dateAdded": "2019-10-02T21:06:00.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers", "expired"]
   },
+  {
+    "name": "IHP",
+    "difficulty": "easy",
+    "description": "Fill out the form, and get some stickers sent to you!",
+    "reference": "https://github.com/digitallyinduced/ihp#ihp-stickers",
+    "image": "https://lh4.googleusercontent.com/0LwDlgsPryo_Du5jkpCgrIPxnO5qNZ3HQtPSt62jYO1kiUBkKDafcgc2MBNF6T_7n_IDsvP0REmNOn3dmv1xvNN00hLnDXu2WNEvjUjvExqy9nbyYOD7uHfct7nW=w376",
```
Comment (aslafy-z) [inline data.json:250]:
```suggestion
    "image": "https://i.imgur.com/lV40hIR.png",
```

**Pull request merged**